src/test/java/racing/CarTest.java

```diff
@@ -1,31 +1,78 @@
 package racing;
 
 import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.CsvSource;
 import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
 import racing.domain.Car;
-import racing.domain.strategies.CarMoveStrategy;
-import racing.utils.Counter;
+import racing.domain.Position;
 
 import java.util.stream.Stream;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 class CarTest {
 
+    @DisplayName("getName()은 자동차 이름을 반환")
+    @ParameterizedTest
+    @ValueSource(strings = {"carA", "carB", "car!", "1car2"})
+    void getName(String name) {
+        Car carA = new Car(name);
+        assertThat(carA.getName()).isEqualTo(name);
+    }
+
+    @DisplayName("getPosition()은 자동차 위치를 반환")
+    @ParameterizedTest
+    @CsvSource(value = {"carA,true,1", "carB,false,0"}, delimiter = ',')
+    void getPosition(String name, Boolean shouldMove, Integer result) {
+        Car car = new Car(name);
+        car.run(() -> shouldMove);
+        assertThat(car.getPosition()).isEqualTo(new Position(result));
+    }
+
+    @DisplayName("자동차A의 위치가 자동차B 보다 앞서면 1, 같으면 0, 뒤쳐지면 -1 반환")
+    @ParameterizedTest
+    @CsvSource(value = {"carA,true,carB,true,0", "carA,true,carB,false,1", "carA,false,carB,true,-1"}, delimiter = ',')
+    void compareToTest(String nameA, Boolean moveA, String nameB, Boolean moveB, Integer result) {
+        Car carA = new Car(nameA);
+        carA.run(() -> moveA);
+        Car carB = new Car(nameB);
+        carB.run(() -> moveB);
```

Comment:
로직을 짤 때 개념적 유사성을 가진 코드를 군집으로 정렬해놓는게 좋습니다.

```suggestion
        Car carA = new Car(nameA);
        Car carB = new Car(nameB);
        
        carA.run(() -> moveA);
        carB.run(() -> moveB);
```